src/components/ui/Chat/ChannelList/ChannelItem.tsx

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React, { HTMLAttributes } from 'react';
+
+import { StyledChannelItem } from './Styled';
+import { useChannelListContext } from './ChannelListContext';
+import PopOver from '../../PopOver';
+import PopOverItem, { PopOverItemProps } from '../../PopOver/PopOverItem';
+import Button from '../../Button';
+import { BaseProps } from '../../Base';
+import { Dots } from '../../icons';
+
+export interface ChannelItemProps extends Omit<HTMLAttributes<HTMLLIElement & HTMLLIElement>, 'css'>, BaseProps {
+  name: string;
+  arn: string;
+  moreOptions?: PopOverItemProps[];
+}
+
+export const ChannelItem = (props: ChannelItemProps) => {
+  const { name, arn, moreOptions } = props;
+  const context = useChannelListContext();
+  const isActive = context.activeChannelArn === arn;
+  return(
+    <StyledChannelItem
+      {...props}
+      className={isActive ? 'isActive' : ''}
```

Comment:
For classes, we typically just add "active" instead of "isActive"

Also, please add "ch-" prefix

Reply:
Yes, I missed the `ch-`. Fixing. I usually use `isActive` so that in the css `element.active` and `element:active` don't get mixed up. I can change it though.